src/main/kotlin/no/nav/familie/ef/iverksett/lagretilstand/LagreTilstandServiceJdbc.kt

```diff
@@ -0,0 +1,92 @@
+package no.nav.familie.ef.iverksett.lagretilstand
+
+import no.nav.familie.ef.iverksett.domene.TilkjentYtelse
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Repository
+import java.util.*
+
+@Repository
+class LagreTilstandServiceJdbc(val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) : LagreTilstand {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    override fun lagreTilkjentYtelseForUtbetaling(behandlingId: UUID, tilkjentYtelseForUtbetaling: TilkjentYtelse) {
+        val sql = "insert into iverksett_resultat values(:behandlingId, null, :tilkjentYtelseForUtbetaling::json, null)"
+
+        val tilkjentYtelseForUtbetalingJson = objectMapper.writeValueAsString(tilkjentYtelseForUtbetaling)
+        val mapSqlParameterSource = MapSqlParameterSource(
+                mapOf(
+                        "behandlingId" to behandlingId,
+                        "tilkjentYtelseForUtbetaling" to tilkjentYtelseForUtbetalingJson
+                )
+        )
+        try {
+            namedParameterJdbcTemplate.update(sql, mapSqlParameterSource)
+        } catch (ex: Exception) {
+            secureLogger.error("Kunne ikke lagre tilkjentYtelseForUtbetaling til basen, behandlingID : ${behandlingId}, tilkjentYtelseForUtbetaling : ${tilkjentYtelseForUtbetaling}")
+        }
+    }
+
+    override fun oppdaterOppdragResultat(behandlingId: UUID, oppdragResultat: OppdragResultat) {
+
+        val sql =
+                "update iverksett_resultat set tilkjentYtelseForUtbetaling = :oppdragResultat::json where behandling_id = :behandlingId"
+
+        val oppdragResultatJson = objectMapper.writeValueAsString(oppdragResultat)
+        val mapSqlParameterSource =
+                MapSqlParameterSource(
+                        mapOf(
+                                "behandlingId" to behandlingId,
+                                "oppdragResultatJson" to oppdragResultatJson
+                        )
+                )
+        try {
+            namedParameterJdbcTemplate.update(sql, mapSqlParameterSource)
+        } catch (ex: Exception) {
+            secureLogger.error("Kunne ikke lagre oppdragResultatJson til basen, behandlingID : ${behandlingId}, oppdragResultatJson : ${oppdragResultatJson}")
```

Comment:
Jeg synes det er enklere å søke i logger hvis man bruker `key=value` i logging, eks
`Kunne ikke lagre oppdragResultatJson til basen, behandling=$behandlingId, oppdragResultatJson= ${oppdragResultatJson}`
Ikke veldig viktig for oppdragResultatJson, men ellers er det fint å ha det slik

Reply:
Enig